? mp_images.patch
? twfy/www/docs/images/mps
? twfy/www/docs/images/mpsL
Index: twfy/www/docs/api/api_getMP.php
===================================================================
RCS file: /repos/mysociety/twfy/www/docs/api/api_getMP.php,v
retrieving revision 1.10
diff -u -r1.10 api_getMP.php
--- twfy/www/docs/api/api_getMP.php	19 Oct 2007 09:24:10 -0000	1.10
+++ twfy/www/docs/api/api_getMP.php	24 Nov 2007 20:28:51 -0000
@@ -36,13 +36,13 @@
 		$row['last_name'], $row['constituency']);
 	if (isset($parties[$row['party']]))
 		$row['party'] = $parties[$row['party']];
-	if (is_file(BASEDIR . IMAGEPATH . 'mpsL/' . $row['person_id'] . '.jpg')) {
+	if (is_file(FILEIMAGEPATH . 'mpsL/' . $row['person_id'] . '.jpg')) {
 		$row['image'] = IMAGEPATH . 'mpsL/' . $row['person_id'] . '.jpg';
-	} elseif (is_file(BASEDIR . IMAGEPATH . 'mpsL/' . $row['person_id'] . '.jpeg')) {
+	} elseif (is_file(FILEIMAGEPATH . 'mpsL/' . $row['person_id'] . '.jpeg')) {
 		$row['image'] = IMAGEPATH . 'mpsL/' . $row['person_id'] . '.jpeg';
-	} elseif (is_file(BASEDIR . IMAGEPATH . 'mps/' . $row['person_id'] . '.jpg')) {
+	} elseif (is_file(FILEIMAGEPATH . 'mps/' . $row['person_id'] . '.jpg')) {
 		$row['image'] = IMAGEPATH . 'mps/' . $row['person_id'] . '.jpg';
-	} elseif (is_file(BASEDIR . IMAGEPATH . 'mps/' . $row['person_id'] . '.jpeg')) {
+	} elseif (is_file(FILEIMAGEPATH . 'mps/' . $row['person_id'] . '.jpeg')) {
 		$row['image'] = IMAGEPATH . 'mps/' . $row['person_id'] . '.jpeg';
 	}
 	$row = array_map('html_entity_decode', $row);
Index: twfy/www/includes/easyparliament/config.php.incvs
===================================================================
RCS file: /repos/mysociety/twfy/www/includes/easyparliament/config.php.incvs,v
retrieving revision 1.7
diff -u -r1.7 config.php.incvs
--- twfy/www/includes/easyparliament/config.php.incvs	6 Nov 2007 13:30:11 -0000	1.7
+++ twfy/www/includes/easyparliament/config.php.incvs	24 Nov 2007 20:28:51 -0000
@@ -56,6 +56,8 @@
 // Web path to the directory where the site's images are kept.
 define ("IMAGEPATH", WEBPATH . "images/");
 
+// Filesystem path to where the images are kept
+define ("FILEIMAGEPATH", BASEDIR . "images/");
 
 // This will be included in data.php. It's an array of page/section metadata.
 define ("METADATAPATH", BASEDIR . "/../includes/easyparliament/metadata.php");
Index: twfy/www/includes/easyparliament/page.php
===================================================================
RCS file: /repos/mysociety/twfy/www/includes/easyparliament/page.php,v
retrieving revision 1.76
diff -u -r1.76 page.php
--- twfy/www/includes/easyparliament/page.php	15 Nov 2007 17:21:26 -0000	1.76
+++ twfy/www/includes/easyparliament/page.php	24 Nov 2007 20:28:52 -0000
@@ -1031,16 +1031,16 @@
 		}
 		print '<p class="printonly">This data was produced by TheyWorkForYou from a variety of sources.</p>';
 		$this->block_start(array('id'=>'mp', 'title'=>$title));
-		if (is_file(BASEDIR . IMAGEPATH . 'mpsL/' . $member['person_id'] . '.jpg')) {
+		if (is_file(FILEIMAGEPATH . 'mpsL/' . $member['person_id'] . '.jpg')) {
 		?>
 						<img src="<?php echo IMAGEPATH;?>mpsL/<?php echo $member['person_id']; ?>.jpg" alt="Photo of <?php echo $member['full_name']; ?>" class="portrait">
-		<? } elseif (is_file(BASEDIR . IMAGEPATH . 'mpsL/' . $member['person_id'] . '.jpeg')) {
+		<? } elseif (is_file(FILEIMAGEPATH . 'mpsL/' . $member['person_id'] . '.jpeg')) {
 		?>
 						<img src="<?php echo IMAGEPATH;?>mpsL/<?php echo $member['person_id']; ?>.jpeg" alt="Photo of <?php echo $member['full_name']; ?>" class="portrait">
-		<? } elseif (is_file(BASEDIR . IMAGEPATH . 'mps/' . $member['person_id'] . '.jpg')) {
+		<? } elseif (is_file(FILEIMAGEPATH . 'mps/' . $member['person_id'] . '.jpg')) {
 		?>
 						<img src="<?php echo IMAGEPATH;?>mps/<?php echo $member['person_id']; ?>.jpg" alt="Photo of <?php echo $member['full_name']; ?>" height="118" class="portrait">
-		<? } elseif (is_file(BASEDIR . IMAGEPATH . 'mps/' . $member['person_id'] . '.jpeg')) {
+		<? } elseif (is_file(FILEIMAGEPATH . 'mps/' . $member['person_id'] . '.jpeg')) {
 		?>
 						<img src="<?php echo IMAGEPATH;?>mps/<?php echo $member['person_id']; ?>.jpeg" alt="Photo of <?php echo $member['full_name']; ?>" height="118" class="portrait">
 		<? } ?>
Index: twfy/www/includes/easyparliament/templates/api/hansard_gid.php
===================================================================
RCS file: /repos/mysociety/twfy/www/includes/easyparliament/templates/api/hansard_gid.php,v
retrieving revision 1.1
diff -u -r1.1 hansard_gid.php
--- twfy/www/includes/easyparliament/templates/api/hansard_gid.php	25 Aug 2006 19:38:31 -0000	1.1
+++ twfy/www/includes/easyparliament/templates/api/hansard_gid.php	24 Nov 2007 20:28:52 -0000
@@ -18,9 +18,9 @@
 		if ($row['htype'] == '12') {
 			if (isset($row['speaker']) && count($row['speaker']) > 0) {
 				$speaker = $row['speaker'];
-				if (is_file(BASEDIR . IMAGEPATH . 'mps/' . $speaker['person_id'] . '.jpeg')) {
+				if (is_file(FILEIMAGEPATH . 'mps/' . $speaker['person_id'] . '.jpeg')) {
 					$row['speaker']['image'] = IMAGEPATH . 'mps/' . $speaker['person_id'] . '.jpeg';
-				} elseif (is_file(BASEDIR . IMAGEPATH . 'mps/' . $speaker['person_id'] . '.jpg')) {
+				} elseif (is_file(FILEIMAGEPATH . 'mps/' . $speaker['person_id'] . '.jpg')) {
 					$row['speaker']['image'] = IMAGEPATH . 'mps/' . $speaker['person_id'] . '.jpg';
 				}
 				$desc = '';
Index: twfy/www/includes/easyparliament/templates/html/hansard_gid.php
===================================================================
RCS file: /repos/mysociety/twfy/www/includes/easyparliament/templates/html/hansard_gid.php,v
retrieving revision 1.13
diff -u -r1.13 hansard_gid.php
--- twfy/www/includes/easyparliament/templates/html/hansard_gid.php	29 Aug 2007 13:50:49 -0000	1.13
+++ twfy/www/includes/easyparliament/templates/html/hansard_gid.php	24 Nov 2007 20:28:53 -0000
@@ -202,12 +202,12 @@
 				$speakername = ucfirst(member_full_name($speaker['house'], $speaker['title'], $speaker['first_name'], $speaker['last_name'], $speaker['constituency']));
 				?>
 				<p class="speaker"><a href="<?php echo $speaker['url']; ?>" title="See more information about <?php echo $speakername; ?>">
-<? if (is_file(BASEDIR . IMAGEPATH . 'mps/' . $speaker['person_id'] . '.jpeg')) { ?>
+<? if (is_file(FILEIMAGEPATH . 'mps/' . $speaker['person_id'] . '.jpeg')) { ?>
                 <img src="<?php echo IMAGEPATH . 'mps/' . $speaker['person_id']; ?>.jpeg" class="portrait" alt="Photo of <?php echo $speakername; ?>" 
 				<?php if (get_http_var('partycolours')) { ?>
                 style="border: 3px solid <?php echo party_to_colour($speaker['party']) ?>;"
                 <?php } ?>
-		><?php } elseif (is_file(BASEDIR . IMAGEPATH . 'mps/' . $speaker['person_id'] . '.jpg')) { ?>
+		><?php } elseif (is_file(FILEIMAGEPATH . 'mps/' . $speaker['person_id'] . '.jpg')) { ?>
 		<img src="<?php echo IMAGEPATH . 'mps/' . $speaker['person_id']; ?>.jpg" class="portrait" alt="Photo of <?php echo $speakername; ?>"><? } ?>
                 <strong><?php echo $speakername; ?></strong></a> <small><?php 
 				$desc = '';
